### PR TITLE
Chore/pdf bugs

### PR DESCRIPTION
**Pull request by ThunderSon**
Don't merge, more issues needs fixing

### CODE REVIEW

**Comment by ThunderSon** [comment]
Don't merge, let me update the appendix structure

File: document/3-The_OWASP_Testing_Framework/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # The OWASP Testing Framework
 
-3.1 [Overview](0-The_OWASP_Testing_Framework.md#overview)
+3.1 [The OWASP Testing Framework](0-The_OWASP_Testing_Framework.md)
```
[the remark this answers is not in the document]
Comment (ThunderSon) [reply document/3-The_OWASP_Testing_Framework/README.md:3]: I'll tackle it, one sec

[the remark this answers is not in the document]
Comment (ThunderSon) [reply document/3-The_OWASP_Testing_Framework/README.md]: should be solved in https://github.com/OWASP/wstg/pull/426/commits/bee7b9925162cf1cf38b4df58929ad7277bcae63

**Comment by kingthorin** [comment]
This one ready now?

**Comment by ThunderSon** [comment]
Yeah, it's done.

**Pull request merged**